packages/tracer/src/Tracer.ts

```diff
@@ -547,6 +547,21 @@ class Tracer extends Utility implements TracerInterface {
     return segment;
   }
 
+  /**
+   * Get the current value of the AWS X-Ray Sampled flag.
+   * 
+   * Utility method that returns the current AWS X-Ray Sampled flag.
+   *
+   * @see https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-traces
+   * 
+   * @returns boolean - `true` if the trace is sampled, `false` if tracing is disabled or the trace is not sampled.
+   */
+  public isTraceSampled(): boolean {
+    if (!this.isTracingEnabled()) return false;
```

Comment:
Should `getRootXrayTraceId` also have a check this like?

Reply:
Possibly yes, it could save some cycles and avoid having to parse a potentially nonexistent variable